servicetalk-http-api/src/main/java/io/servicetalk/http/api/DefaultHttpExecutionStrategyInfluencer.java

```diff
@@ -17,16 +17,11 @@
 
 import io.servicetalk.transport.api.ExecutionStrategyInfluencer;
 
-import static io.servicetalk.http.api.DefaultHttpExecutionStrategy.OFFLOAD_ALL_STRATEGY;
-
-/**
- * A factory for HTTP filters.
- */
-public interface HttpFilterFactory extends ExecutionStrategyInfluencer<HttpExecutionStrategy> {
+interface DefaultHttpExecutionStrategyInfluencer extends ExecutionStrategyInfluencer<HttpExecutionStrategy> {
```

Comment:
Many of the classes that implement this interface will be public and any similar user classes will also want to use this interface. A similar example would be the Java `StringBuilder` and `StringBuffer` classes which extend the non-public `AbstractStringBuilder` class which receives constant complaints.

In general, public classes and interfaces should have only public ancestors.

I am also not convinced by the name change. It seems too broad.

Reply:
On further thought it seems reasonable as long as users won't need to recreate the same interface themselves.